src/components/pages/ServiceReleaseProcess/components/ServiceDetails.tsx

```diff
@@ -48,6 +51,7 @@ export default function ServiceDetails() {
     refetchOnMountOrArgChange: true,
   }).data
   const [fetchDocument] = useFetchDocumentMutation()
+  const [image, setActualImage] = useState<string>('')
```

Comment:
by convention destructured useState variables should always be named
`[someValue, setSomeValue] = useState...`

see sonar finding: https://sonarcloud.io/project/issues?sinceLeakPeriod=true&issueStatuses=OPEN%2CCONFIRMED&pullRequest=992&id=eclipse-tractusx_portal-frontend&open=AZExprbVdf87lRO01BrH